### DISCUSSION TITLE
Importing manual settings from armoury crate

### DISCUSSION THREAD

**Discussion by MrXT21** [seerge/g-helper#1149] [Ideas]
Could you implement a feature that allows importing manual settings from armoury crate? Iv found it very hard to redo manual settings I have in armoury crate on ghelper due to the fact that armoury crates fan speeds are set in % while ghelper has it set in the fans RPM.

Perhaps a setting that switches ghelper from displaying RPM to % so that manually recreating settings is simplier? I found silent is easy to recreate since it requires no fan and i get lower temps but when doing gaming profiles my settings are way cooler in armoury crate than in ghelper because i just cant recreate the same settings I had properly.

**Comment by seerge** [comment]
AC is proprietary software, I don't plan any integrations with it.

**Reply by MrXT21** [reply]
Okay, and the second part?

**Reply by seerge** [reply]
@MrXT21 for second part, if I will add % in a same way they are shown in AC - i.e. 0 - 10 - 20 ... 100. It's simply incorrect scale, as 0% for Asus means OFF and 1% immediately means 1800RPM usually (or ~ 30% in reality).

It will cause a lot of rage on reddit and here in comments, saying that % are incorrect.

You can just look at horizontal lines on chart, they are actually even now same as % lines in AC :)

**Reply by MrXT21** [reply]
Iv tried that, the fan curve looks identical and yet on armoury crate my cpu temp is stable at 79c while gaming and ghelper its at 87c. Couldnt you put a toggle in settings to display in % or RPM so theres no complaints from anyone. Assuming its accurate it would help anyone trying to recreate settings from armoury crate.

**Reply by seerge** [reply]
Well, your temperature would depend like on 10 different factors :) main one would be power consumed by CPU itself and other things.

**Reply by seerge** [reply]
@MrXT21 in this build I have made Y-Axis on all charts clickable, it will toggle RPM / % 

But I doubt your temperature difference is cause of fan curve :)

[GHelper.zip](https://github.com/seerge/g-helper/files/12412042/GHelper.zip)

**Reply by MrXT21** [reply]
My manual gaming profile is capped at 25w on both armoury crate and ghelper.

Thank you, ill give this a shot and see how it goes. Your hard work on this project is very much appreciated!!
